src/main/webapp/itempagescript.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Retrieves ItemPageData and forms page using other functions.
+ */
+async function loadItemPage() {
+  const itemId = getItemId();
+  console.log(itemId);
+  if (itemId != null) {
+    fetch(`/itempagedata?itemId=${itemId}`)
+        .then((response) => response.json())
+        .then((ItemPageData) => {
+          createSelectedItemCard(ItemPageData.item);
+          getItemPageComments(ItemPageData.comments);
+        })
+  } else {
+    console.log('ItemId is null');
+  }
+}
+
+
+function createSelectedItemCard(entertainmentItem) {
+  const card = $('<div class="card bg-light"></div>');
+  card.append(
+      $('<img class="card-img-top" src="' + entertainmentItem.imageUrl + '">'));
+  const cardBody = $('<div class="card-body"></div>');
+  cardBody.append(
+      $('<h5 class="card-title">' + entertainmentItem.title + '</h5>'));
+  cardBody.append(
+      $('<p class="card-text">' + entertainmentItem.description + '</p>'));
+
+  card.append(cardBody);
+  const itemContainer = $('#item-container');
+  itemContainer.append(card);
+}
+async function getItem() {
+  const itemId = getItemId();
+  console.log(itemId);
+  const response = await fetch(`/itempagedata?itemId=${itemId}`);
+  const selectedItem = await response.json();
+  createSelectedItemCard(selectedItem);
```

Comment:
Add a try-catch block to check when the fetch request fails and output an error message accordingly.